src/test/java/bio/terra/endpoints/JavaHTTPRequestTests.java

```diff
@@ -0,0 +1,125 @@
+package bio.terra.endpoints;
+
+import bio.terra.common.category.CLIIntegrated;
+import bio.terra.context.Login;
+import org.junit.*;
```

Comment:
Nit: we just don't _do_ asterisks! :)

Reply:
Done. Good catch. I've also updated my IntelliJ settings so hopefully this doesn't happen automatically any more.